index.html

```diff
@@ -2072,20 +2072,18 @@ <h3>Extensibility</h3>
           <h4>Semantic Interoperability</h4>
 
           <p>
-This specification endeavors to enable both the JSON and JSON-LD syntaxes to
-be semantically compatible with one another without the JSON implementations
-needing to process the documents as JSON-LD. To achieve this, the
-specification imposes the following additional restrictions on both syntaxes:
+This specification ensures that "plain" JSON and JSON-LD syntaxes are semantically
+compatible without requiring JSON implementations to use a JSON-LD processor.
+To achieve this, the specification imposes the following additional restrictions,
+regarding the use of <code>@context</code>:
           </p>
 
           <ul>
             <li>
-JSON-based processors MUST process the <code>@context</code> <a>property</a>,
-ensuring the expected values exist in the expected order for the
-<a>credential</a> type being processed. It is advised that the expected order
-of values of the <code>@context</code> <a>property</a> should be defined by
-at least a human-readable extension specification and preferably by a
-machine-readable specification as well.
+JSON-based processors MUST process the <code>@context</code> <a>property</a>
```

Comment:
Agree with the simplification here and moving the text to a paragraph below.

Reply:
I have just been designing the parser for our JSON implementation, and it seems to me that the only reason that I need to process the @context is to ensure that the terms (aliases that are used for URIs) in the body of the VC are pointed to by the @context values i.e. that no terms are present in the VC that are not pointed to by the @context values. Of course in our implementation a human will have read the contents of the @context web pages that are pointed to and will have configured the alias terms into our JSON parser so that it will be happy to encounter either the full URIs or the short form aliases. 

It might be useful to tell JSON implementors that they do not need to define new @context values themselves, since they can use full URIs for all their new VC terms if they want to.

Reply:
I question whether it would be good to do this as this is not a common practice for HTTP APIs that use JSON (or JSON documents in general).